.prettierignore

```diff
@@ -5,6 +5,7 @@ build
 node_modules
 snark-params
 vue-app/gundata/
+vue-app/src/graphql/API.ts
```

Comment:
Ignoring autogenerated file

Reply:
Should we add this to .gitignore as well?

Reply:
No, actually we want this file because it is going to give us all the api to our graphql sdk and will give us all the typing information so that you have nice autocompletes and typecheckings.

I'm ignoring it from prettier because when you generate it, it will use some custom styling.